site/reference/schema/kptfile/kptfile.json

```diff
@@ -1,105 +1,125 @@
 {
-  "$schema": "http://json-schema.org/draft/2019-09/schema#",
-  "type": "object",
-  "anyOf": ["#/definitions/Kptfile"],
```

Comment:
I think we will still need this "anyOf" block in order for language server support to work properly

Reply:
Yeah. That should be manual step before transferring the schema to language server. In the meanwhile we will try to figure out if there is automated way to a add this as part of generation.